ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/provision/cucushift-installer-rehearse-aws-ipi-proxy-provision-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-proxy-provision
+  steps:
+  - chain: ipi-conf-aws-proxy
+  - chain: ipi-install
+  documentation: |-
+    Create an IPI cluster on AWS for QE e2e tests.
```

Comment:
Perhaps refactor the line to contain `proxy` ?

Reply:
I'm not quite understand the `refactor` here, can you explain more about it? thanks, actually, this is the original workflow that is being used by developer https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/aws/pre/proxy/ipi-aws-pre-proxy-chain.yaml